scripts/get-prs

```diff
@@ -20,13 +36,93 @@ import sys
 
 from github import Github
 
-test_start_re = "[Tt]est with:?"
-repo_ref_re = "([a-zA-Z0-9_-]+/[a-zA-Z0-9_-]+#[0-9]+)"
-pr_re = re.compile(repo_ref_re)
-repo_sep_re = "([, ]+(and)?[ ]*)"
-test_re = re.compile(f"{test_start_re}[ ]*({repo_ref_re}{repo_sep_re}?)+")
+# Start with a few simple regex'es and re-use them to build more complex ones.
```

Comment:
This should not be a tutorial
```suggestion
```

Reply:
It's just for the first round of reviews ;)